### PR TITLE
Notice management page in the admin dashboard AIRAVATA-1962 and  AIRAVATA-3071

### PR DESCRIPTION
**Pull request by shivamriky**
- Add a new notice
- Edit a notice
- Delete a notice
- Notification dropdown
- Persist read notifications after the user logs out and only show the number of unread notifications on the notification icon

### CODE REVIEW

**Comment by shivamriky** [comment]
@machristie I have completed all the changes and added the dropdown menu for the notifications. Please let me know when can we do a code review.

File: django_airavata/templates/base.html [hunk 1]
```diff
@@ -91,6 +92,54 @@
     animation-delay: 0.9s;
   }
 
+  .widget-notifications{max-height:300px;overflow-y:auto;}
```
Comment (machristie) [inline django_airavata/templates/base.html]: Let's move this css into main.scss.

File: django_airavata/apps/admin/static/django_airavata_admin/src/components/notices/NoticeEditor.vue [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+</style>
+<script>
+import { models } from "django-airavata-api";
+import { mixins, utils } from "django-airavata-common-ui";
+import { Datetime } from 'vue-datetime';
+import moment from "moment";
+import 'vue-datetime/dist/vue-datetime.css'
+
+export default {
+  name: "noitce-editor",
```
[the remark this answers is not in the document]
Comment (shivamriky) [reply django_airavata/apps/admin/static/django_airavata_admin/src/components/notices/NoticeEditor.vue]: I wonder how was it recognized in the noticemanagement container with the spelling error. I will fix it.

File: django_airavata/templates/base.html (hunk 1 above)
Comment (shivamriky) [reply django_airavata/templates/base.html]: Fixed,

File: django_airavata/apps/api/views.py [hunk 3]
```diff
@@ -1387,6 +1387,53 @@ def get(self, request, format=None):
...
+        self.request.airavata_client.updateNotification(self.authz_token, notification)
+
+
+class AckNotificationViewSet(APIView):
+
+
+    def get(self, request, format=None):
+        if 'id' in request.GET:
+            notification_id = request.GET['id']
+            print(notification_id)
```
[the remark this answers is not in the document]
Comment (shivamriky) [reply django_airavata/apps/api/views.py]: Fixed.

File: django_airavata/apps/admin/static/django_airavata_admin/src/components/notices/NoticeEditor.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (shivamriky) [reply django_airavata/apps/admin/static/django_airavata_admin/src/components/notices/NoticeEditor.vue]: Fixed.

**Pull request merged**